ethereum/.env.optimism.sepolia.testnet

```diff
@@ -0,0 +1,17 @@
+# Optimism on Sepolia testnet read only env
+# Rename to .env to use with truffle migrations
+
+# Wormhole Core Migrations
+INIT_SIGNERS=["0x13947Bd48b18E53fdAeEe77F3473391aC727C638"]
+INIT_CHAIN_ID=10005
+INIT_GOV_CHAIN_ID=0x1
+INIT_GOV_CONTRACT=0x0000000000000000000000000000000000000000000000000000000000000004
+INIT_EVM_CHAIN_ID=11155420
+
+# Bridge Migrations
+BRIDGE_INIT_CHAIN_ID=10005
+BRIDGE_INIT_GOV_CHAIN_ID=0x1
+BRIDGE_INIT_GOV_CONTRACT=0x0000000000000000000000000000000000000000000000000000000000000004
+# Not sure if there should be a WETH.
+BRIDGE_INIT_WETH=0x0000000000000000000000000000000000000000
```

Comment:
I think for OP stack chains, the native bridged WETH is at `0x4200000000000000000000000000000000000006`

Reply:
Updated the contract to reflect this.